### ISSUE TITLE
GHC 9 Support

### ISSUE THREAD

**Issue by parsonsmatt** [lykahb/ory-hydra-client#3]
The current version of this package does not support GHC 9. Specifically getting errors related to the `eta` expansion stuff. Error here:

```
ory-hydra-client        > [ 1 of 11] Compiling ORYHydra.LoggingKatip
ory-hydra-client        >
ory-hydra-client        > /tmp/stack-ab9523d3b4361695/ory-hydra-client-1.9.2/lib/ORYHydra/LoggingKatip.hs:57:32: error:
ory-hydra-client        >     • Couldn't match type: LG.KatipT m0 a0 -> m0 a0
ory-hydra-client        >                      with: forall a. LG.KatipT m a -> m a
ory-hydra-client        >       Expected: LogContext -> LogExec m
ory-hydra-client        >         Actual: LG.LogEnv -> LG.KatipT m0 a0 -> m0 a0
ory-hydra-client        >     • In the expression: LG.runKatipT
ory-hydra-client        >       In an equation for ‘runDefaultLogExecWithContext’:
ory-hydra-client        >           runDefaultLogExecWithContext = LG.runKatipT
ory-hydra-client        >     • Relevant bindings include
ory-hydra-client        >         runDefaultLogExecWithContext :: LogContext -> LogExec m
ory-hydra-client        >           (bound at lib/ORYHydra/LoggingKatip.hs:57:1)
ory-hydra-client        >    |
ory-hydra-client        > 57 | runDefaultLogExecWithContext = LG.runKatipT
ory-hydra-client        >    |                                ^^^^^^^^^^^^
```

It appears that this PR added support for GHC 9: https://github.com/OpenAPITools/openapi-generator/pull/10400

**Comment by lykahb** [comment]
Thanks! I'll update the client.

**Comment by lykahb** [comment]
Published 1.10 on Hackage. It works with the nightly stack resolver and GHC 9.0.1.

**Comment by parsonsmatt** [comment]
thank you!
